src/script/telemetry/calling/CallSetupTimings.ts

```diff
@@ -70,8 +92,8 @@ class CallSetupTimings {
 
     this._steps_order().forEach(step => {
       if (this.hasOwnProperty(step)) {
-        const placeholder_key = Array.from(Math.max(26 - step.length, 1)).join(' ');
-        const placeholder_value = Array.from(Math.max(6 - this[step].toString().length, 1)).join(' ');
+        const placeholder_key = new Array(Math.max(26 - step.length, 1)).join(' ');
+        const placeholder_value = new Array(Math.max(6 - this[step].toString().length, 1)).join(' ');
```

Comment:
`Array.from` actually needs an array-like parameter. What we want to achieve here is a new array with a specified length and fill it.

Reply:
Then how about `' '.repeat()`?